big_tests/tests/mongooseimctl_SUITE.erl

```diff
@@ -193,8 +193,9 @@ init_per_suite(Config) ->
     Config1 = ejabberd_node_utils:init(Node, Config),
     Config2 = escalus:init_per_suite([{ctl_auth_mods, AuthMods},
                                       {roster_template, TemplatePath} | Config1]),
-    dynamic_modules:ensure_modules(domain_helper:host_type(), [{mod_last, []}]),
-    dynamic_modules:ensure_modules(domain_helper:secondary_host_type(), [{mod_last, []}]),
+    dynamic_modules:ensure_modules(domain_helper:host_type(), [{mod_last, #{iqdisc => one_queue}}]),
```

Comment:
Maybe use `config_parser_helper:default_mod_config(mod_last)` in all these places where the module defaults are provided, might reuse code and simplify using defaults later.